.github/workflows/build_tests.yml

```diff
@@ -0,0 +1,84 @@
+# Contains jobs corresponding to build tests (such as verifying that certain targets build, including the app itself).
+
+name: Build Tests
```

Comment:
Can we say it `Bazel Build Test`, as we will be running the tests on Bazel or is it just Bazel Building in this file?

Reply:
ah, we are running Bazel test in `.github/workflows/unit_tests.yml`, so I guess we can rename this file which gives a sense that this file is trying to build apk using bazel.

Reply:
The idea for this file was that it hold all build tests (e.g. tests that are pass/fail based on whether a build succeeds rather than the runtime). This may be useful as a location to add future tests where building is much faster than running to get an early-failure indication for those tests without waiting for the full runtime to complete (such as potentially end-to-end tests).